src/Equinor.ProCoSys.IPO.Infrastructure/Migrations/20210217091145_UniqueConstraintOnOid.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Equinor.ProCoSys.IPO.Infrastructure.Migrations
+{
+    public partial class UniqueConstraintOnOid : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateIndex(
```

Comment:
NB! This will cause migration problems in a DB if non-unique OID's found. Be prepared, I guess this WILL happen.

Reply:
Thanks, if I check all db before merging